docs/configuration.asciidoc

```diff
@@ -239,5 +239,22 @@ By default the agent can only send (up to) `20` transactions every `30000` milli
 A number between `0.0` and `1.0` that specifies the sample rate of transactions, by default all transactions are sampled.
 
 
+[float]
+[[central-config]]
+==== `centralConfig`
+
+* *Type:* Boolean
+* *Default:* `false`
+
+This option activates APM Agent Configuration via Kibana.
+If set to `true`, the agent will fetch the central configuration via the APM Server on the initialization phase.
```

Comment:
Better to say it as the agent starts polling for configuration during the initialisation phase and the config is stored in the

Reply:
Personally I'm not a fan of `polling` as it is ambiguous and implies repeated fetching, I think using fetch is more clear.

Reply:
Yeah polling was wrong, i rephrased it a bit.

Reply:
I think what you have works. I also think @vigneshshanmugam's suggestion would work. For fun, if I were writing this, here's how I would phrase it:
```
If set to `true`, the agent starts fetching configurations via the APM Server during the initialization phase.
These central configurations are cached in `sessionStorage`, and will not be fetched again until
the session is closed and/or `sessionStorage` is cleared. In most cases, this means when the tab/window of the page is closed.
```